apps/src/code-studio/pd/application_dashboard/detail_view_response.jsx

```diff
@@ -26,9 +26,41 @@ Question.propTypes = {
 export default class DetailViewResponse extends React.Component {
   static propTypes = {
     question: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
+    questionId: PropTypes.string,
     answer: PropTypes.oneOfType([PropTypes.string, PropTypes.array, PropTypes.bool]),
-    layout: PropTypes.oneOf(['lineItem', 'panel']).isRequired
-  };
+    layout: PropTypes.oneOf(['lineItem', 'panel']).isRequired,
+    score: PropTypes.string,
+    possibleScores: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
+    editing: PropTypes.bool,
+    handleScoreChange: PropTypes.func
+  }
+
+  renderScoreOptions = () => {
+    return this.props.possibleScores.map((score, i) => {
+      return (
+        (
+          <option value={score} key={i}>
+            {score}
+          </option>
+        )
+      );
```

Comment:
These can both be expressions:
```jsx
renderScoreOptions = () => this.props.possibleScores.map((score, i) => (
  <option value={score} key={i}>
    {score}
  </option>
);
```

Also since this isn't passed to a callback, it doesn't need to bind this and could also be written as a class method. No strong preference:

```jsx
renderScoreOptions() {
  return this.props.possibleScores.map((score, i) => (
    <option value={score} key={i}>
      {score}
    </option>
  );
}
```